webcompat/views.py

```diff
@@ -275,14 +276,19 @@ def create_issue():
 @app.route('/issues/<int:number>')
 @cache_policy(private=True, uri_max_age=0, must_revalidate=True)
 def show_issue(number):
+    issue_body_json, issue_body_status_code, issue_body_headers = api.proxy_issue(number)
```

Comment:
This will help us also to treat the [PR and deleted issues](https://github.com/webcompat/webcompat.com/issues/1311) in a more elegant way.

Reply:
Sorry @karlcow, but I don't know exactly what you mean.
I read the issue and I saw you gave a few solutions:

> - We create a relevant generic body message the same way we do for our 404 message.
> - So all of these can be put in a specific milestone, called `ignore` and it's another way to identify them.

What do you think it could be a good solution here? Thanks.

Reply:
Sorry. This is just a side comment to say that server side rendering will help us to solve #1311 in a simpler way that all the acrobatic solutions we were trying to do. This will be straightforward (if we decide to go that way).

Reply:
So basically, you can ignore for now. And we can solve it later if we deploy this.

Reply:
Okay! :)